.style.yapf

```diff
@@ -1,5 +1,5 @@
 [style]
-based_on_style = google
+based_on_style = pep8
```

Comment:
Hmm...Why switch to this? I think this is a much bigger change, can we just switch to google?

Reply:
But it seems that the behavior is different

Reply:
I reseted to the bump for 0.43.0 version.